packages/react/src/domain/constants/solanaWalletInjections.ts

```diff
@@ -0,0 +1,10 @@
+export const SOLANA_WALLET_INJECTIONS = [
+  'phantom',
+  'solflare',
+  'coinbaseSolana',
+  'braveSolana',
+  'sollet',
+  'glow',
+  'slope',
+  'solana',
+];
```

Comment:
I feel like we defo have this on BE or FE

Reply:
@00MB moved on sdk